tests/scenarios/signing/ExportCertificate.ps1

```diff
@@ -0,0 +1,24 @@
+param (

```

Comment:
Should add a comment block at the beginning describing the file and parameters to keep in parity with other scripts in this code base.